indy_node/test/api/helper.py

```diff
@@ -184,3 +184,28 @@ def sdk_write_schema(looper, sdk_pool_handle, sdk_wallet_steward):
     request = looper.loop.run_until_complete(build_schema_request(identifier, schema_json))
     return schema_json, \
            sdk_get_reply(looper, sdk_sign_and_submit_req(sdk_pool_handle, sdk_wallet_steward, request))[1]
+
+
+def sdk_write_schema_and_check(looper, sdk_pool_handle, sdk_wallet_steward,
+                               multi_attribute=[], name="", version=""):
+    _, identifier = sdk_wallet_steward
+
+    if multi_attribute:
+        _, schema_json = looper.loop.run_until_complete(
+            issuer_create_schema(
+                identifier, name,
+                version, json.dumps(multi_attribute)
+            ))
+    else:
```

Comment:
Why do we need this else here?

Reply:
This is for the functions that relied on this helper before my modifications.

Reply:
I was incorrect. Yes I believe this if can be removed